enemies.js

```diff
@@ -77,14 +77,178 @@ class Goomba {
     };
 };
 
-class Koopa {
+class KoopaParatroopaRed {
```

Comment:
Having two classes for each color seems to cause a lot of code duplication. While this is an optional suggestion, making the structure like the original koopa, where you take a color parameter, and do checks for that to determine differing behavior would make the code more clean imo.